content/docs/jsx-in-depth.md

```diff
@@ -12,32 +12,32 @@ redirect_from:
   - "docs/jsx-in-depth-zh-CN.html"
   - "docs/jsx-in-depth-ko-KR.html"
 ---
-
-Fundamentally, JSX just provides syntactic sugar for the `React.createElement(component, props, ...children)` function. The JSX code:
+<!-- This jsx-in-depth.md` is translated by dummyeuy (Muhammad Ghalib) -->
```

Comment:
Please remove this line.

Reply:
line removed

Reply:
> Please remove this line.

dear @resir014  I've applied the corrections you gave.
Latest commit on 
https://github.com/dummyeuy/id.reactjs.org/commit/66e906a993e25e9e9e48f1994102bbaa622d246b

Please kindly review it